### PR TITLE
Questions and reflections

### PR DESCRIPTION
**Pull request by XianQu2016**
1. The order of emphasis and header: figured out that header code should go first and then emphasis
2. This assignment is quite interesting. I went back to the installation steps before to help myself remember and then start new steps.Also, besides steps on Github website of Markdown, it could also work from R Markdown.

### CODE REVIEW

**Comment by jmausolf** [comment]
## Evaluation HW-01 for Xian Qu

| Topic | Excellent | Satisfactory | Needs Work |
| --- | --- | --- | --- |
| **Coding style** |        x  |     |             |
| **Coding strategy** |     x   |        |        |
| **Presentation: graphs** |      |     |       |
| **Presentation: tables** |       |       |     |
| **Achievement, creativity** |x      |      |         |
| **Ease of access** | x |                 |             |

Remarks:
- Please put your name *Xian Qu* in the commit message. This helps us easily know whose homework is whose as github usernames are often ambiguous.
- I love the photo you used. Creative.
- Otherwise, you effectively used four of the five style elements, italics, photo, headers, and list. Good job.
- You may want to include a .gitignore or at least use `git status` to check which files you stage before committing them.
- Great use of git. Proved you can do the workflow.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# hw01
-Repository for first CFSS homework assignment
+# **Introduction**
+
```
Comment (yanningcui) [inline README.md:2]: good use of header!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+## *Name*
+Xian Qu
+
+## *Program*
+MAPSS
+
+## *Personality*
```
Comment (yanningcui) [inline README.md:9]: good use of bold!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
...
+1. Outgoing
+2. Hardworking
+3. Open-minded:
+   - towards people
+   - towards life
+
```
Comment (yanningcui) [inline README.md:15]: good use of list

File: README.md [hunk 4]
```diff
@@ -1,2 +1,21 @@
...
+## *Picture*
+![Image of Snoopy](https://s-media-cache-ak0.pinimg.com/originals/00/45/96/004596e8d225dae6edb2f727cc33fe41.png)
+
```
Comment (yanningcui) [inline README.md:18]: successfully linked to a cute picture! well done!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,21 @@
...
+##*Questions and Reflections*
+1. The order of emphasis and header: figured out that header code should go first and then emphasis
+2. This assignment is quite interesting. I went back to the installation steps before to help myself remember and then start new steps.Also, besides steps on Github website of Markdown, it could also work from R Markdown.
```
Comment (yanningcui) [inline README.md:21]: Actually, if  you use R Markdown, you will get a table of information (tilte, author, date...) on top of the page. (just like my hw01)
You can also try insertting a web link (if you don't count the link of the picture)

**Review by yanningcui — COMMENTED** [review]
This homework meets the requirement. I learn the personality of XianQu, which is great! It would be better if Xian could add more words in her self introduction, and also insert a web link. Good job!

**Comment by incipamus** [comment]
Peer Review for Xian
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Hi Xian. Overall, this is a nice job. 
* The requirements for the assignment are well satisfied in your README.md
* I like your big picture. This is a question that I couldn't find an answer yet, but do you how to resize the image?
* FYI, I found you can put a note to explain your code using "[//]: # ( )". Hope this helps.

**Pull request closed without merging**